### ISSUE TITLE
Bug 54111342: [React Native Gallery -> Home]: The text present inside 'Get started/React Native/Windows Design/GitHub...' etc links are getting truncated after applying text scaling to 200%.

### ISSUE THREAD

**Issue by Yajur-Grover** [microsoft/react-native-gallery#503]
Build Details:
OS Version: Dev (OS Build 27695.1000)
App: React Native Gallery 
App Version: 1.0.18.0
React Native Windows Version: 0.75.0

AT: Text Scaling  
Resolution: 1280*768

Pre-Requisite:
Text Scaling (Settings->Accessibility->Text size-> Drag the slider to 200% ->Apply))

Repro-Steps:
Launch the React Native Gallery app.
Navigate to ‘Home' button present in left navigation pane and invoke it.
Now observe the text present inside the 'Get started/React Native/Windows Design/GitHub...' etc links after applying text scaling.
Actual Result:
The text present inside 'Get started/React Native/Windows Design/GitHub...' etc links are getting truncated after applying text scaling to 200%.

Note: Same issue is observed in the entire application.

Expected Result:
Text present inside 'Get started/React Native/Windows Design/GitHub...' etc links should be visible clearly to the user without any data loss and truncation.

User impact:

Users with low vision will not be able to consume the content of the button if it is not visible in Text scaling mode.

Link to ADO bug: https://dev.azure.com/microsoft/OS/_workitems/edit/54111342
